src/app/content/components/Seats.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Flex, Heading, Progress } from '@chakra-ui/react';
+import { Seat } from '../../../fetchers';
+
+export interface SeatsProps {
+  seat?: Seat;
+}
+
+export function SeatInfo({ seat }: SeatsProps): JSX.Element {
+  const seatsPercent = seat === undefined ? undefined : (seat.seats.actual / seat.seats.capacity) * 100;
+  const waitlistPercent =
+    seat === undefined ? undefined : (seat.waitListSeats.actual / seat.waitListSeats.capacity) * 100;
+  return (
+    <Flex my="2" direction={{ base: 'column', md: 'row' }}>
+      <Box width={{ base: '100%', md: '50%' }}>
+        <Flex justifyContent="space-between" my="1">
+          <Heading as="h6" size="sm">
+            Seats
+          </Heading>
+          <Heading as="h6" size="sm">
+            {seat === undefined ? '' : `${seat.seats.actual} / ${seat.seats.capacity}`}
+          </Heading>
+        </Flex>
+        {seat !== undefined ? (
+          <Progress value={seatsPercent} colorScheme={seat.seats.actual >= seat.seats.capacity ? 'red' : 'green'} />
```

Comment:
wouldnt it be better if we show them "how many seats are remaining" along with the progress bar